src/components/UI/BreadCrumbs.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="breadcrumbs">
+    <button
+      class="breadcrumbs__link breadcrumbs__link--btn"
+      @click="$router.push('/')"
+    >
+      <div>
+        <img src="@/assets/img/arrow-back.svg" />
+      </div>
+      <div>Back</div>
+    </button>
+    <p
+      class="breadcrumbs__link breadcrumbs__link--crumb"
+      @click="secondLevel ? $router.go(-1) : $router.go(0)"
```

Comment:
It's pretty interesting idea to use `$router.go(-1)` but I'm not sure it it is always a good idea. Consider movie detail page: you can probably get there from not only from from movies catalogue, but also from homepage,  from screenings page or even from account page (reservations). Maybe it would be safer to actually pass the route here explicitly as a prop together with the label (`secondLevel`)?

Reply:
Based on you feedback:

I have changed back button to be have like a back button - going back to previous view

And I have changed first tier to take back path as a prop aso It can go back one tier down [ for example Movies or Scrrenings]

thank you so much!! 😸